app/db_models/address.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Column, ForeignKey, Integer, Numeric, String
+from sqlalchemy.orm import relation
+
+from app.db.base_class import Base
+from geoalchemy2.types import Geography
+
+
+class Address(Base):
+    """To store the addresses of user.
+    
+    `tag` can be Home, Work, Other etc.
+    """
+
+    __tablename__ = 'addresses'
```

Comment:
Should be singular

Reply:
@iamareebjamal I have followed this same convention in some other tables also. Why is this wrong?

Reply:
https://stackoverflow.com/a/5841297/3309666

Addresses sounds weirder than Address